packages/py-api/py_api/utilities/logging.py

```diff
@@ -32,10 +35,36 @@
     },
 }
 
+# Overwrite uvicorn's default logging config in order to be able to log INFO level messages

```

Comment:
Let's rephrase this to:

Overwrites uvicorn's default logging config in order to enable INFO level when using our own logger handlers